patches/net/minecraft/client/renderer/LevelRenderer.java.patch

```diff
@@ -219,6 +237,14 @@
 +            this.globalBlockEntities.forEach(blockEntityConsumer);
 +        }
 +    }
++
++    /**
++     * Request outline effect post-processing to be enabled for the current frame. Must be called before block entities
++     * are done rendering, ideally early during the frame
++     */
++    public void requestOutlineEffect() {
++        this.outlineEffectRequested = true;
++    }
```

Comment:
In my opinion, this (and its corresponding field) should have the `Neo:` designator in order to clearly distinguish them from vanilla code. 

This is particularly important when using Parchment mappings, since the presence of javadocs may make it seem like a vanilla method and field.